python-lib/hubspot/contacts.py

```diff
@@ -0,0 +1,43 @@
+import dataiku
+import pandas as pd, numpy as np
+from dataiku import pandasutils as pdu
+import json, time, requests
+import urllib
+from pandas.io.json import json_normalize
+
+def get_contact_properties(apikey):
+    try:
+        r = requests.get("https://api.hubapi.com/properties/v1/contacts/properties?", params = {'hapikey': apikey})
+    except:
+        if (r.status_code != 200 and method == 'get'):
+            raise Exception('API error when calling {}, error code {}. Returned response : {}'.format(r.url, r.status_code, r.json()))
+    response_dict = r.json()
+    list_contacts_properties = [x[u'name'] for x in response_dict]
+    return list_contacts_properties
+
+def get_contacts(apikey, properties_type, list_input):
+    limit = 100
+    if properties_type == 'Standard':
+        parameter_dict = {'hapikey': apikey, 'count': limit, 'property': contact_properties}
+    elif properties_type == 'All':
+        contact_properties = get_contact_properties(apikey)
+        parameter_dict = {'hapikey': apikey, 'count': limit, 'property': contact_properties}
+    elif properties_type == 'Custom':
+        contact_properties = list_input
+        parameter_dict = {'hapikey': apikey, 'count': limit, 'property': contact_properties}   
+    has_more = True
+    counter = 0
+    while has_more:
```

Comment:
There is a lot of similarities between get_companies and this method, factorization can be made.

Reply:
The pagination is different (yes, I know...) so maybe we can keep it this way?

Reply:
What is different can be parameterized (count vs limit, property vs properties...)